webapp/app.py

```diff
@@ -890,6 +890,20 @@ def takeovers_index():
 )
 core_als_autils_docs.init_app(app)
 
+smartstart_url_prefix = "/core/smartstart/guide"
```

Comment:
It's difficult to understand what they want from the brief but the word replace would suggest they want the guide at this URL. Just strange as we don't have a smartstart section on the site.

Reply:
I understood it as they want to replace the existent URL? Not the current smart start guide, because that one will receive a redirect? But I can talk to them see if that's what they mean.

Reply:
Updated to `/core/services/guide`